pkg/cmd/server/origin/admission/plugin_initializer.go

```diff
@@ -136,16 +137,18 @@ func NewPluginInitializer(
 	openshiftPluginInitializer := &oadmission.PluginInitializer{
 		ProjectCache:                 projectCache,
 		OriginQuotaRegistry:          quotaRegistry,
-		JenkinsPipelineConfig:        options.JenkinsPipelineConfig,
 		RESTClientConfig:             *privilegedLoopbackConfig,
 		ClusterResourceQuotaInformer: informers.GetInternalOpenshiftQuotaInformers().Quota().InternalVersion().ClusterResourceQuotas(),
 		ClusterQuotaMapper:           clusterQuotaMappingController.GetClusterQuotaMapper(),
 		RegistryHostnameRetriever:    registryHostnameRetriever,
 		SecurityInformers:            informers.GetInternalOpenshiftSecurityInformers(),
 		UserInformers:                informers.GetOpenshiftUserInformers(),
 	}
+	jenkinsPipelineConfigInitializer := &jenkinsbootstrapper.PluginInitializer{
+		JenkinsPipelineConfig: options.JenkinsPipelineConfig,
+	}
```

Comment:
why isn't it just another openshiftPluginInitializer?

Reply:
> why isn't it just another openshiftPluginInitializer?

Every other plugin initializer is used in more than one location or by more than one process.  Yours was a shortcut to avoid writing an admission config struct and is only used on one spot for one server.